WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table do %>
+  <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
     <thead>
       <tr>
-        <th class="name">Name</th>
-        <th class="wca-id">WCA ID</th>
-        <th class="country">Citizen of</th>
+        <th class="name" data-sortable="true" data-field="name">Name</th>
```

Comment:
Yes, data-sort-name indicates the default field to sort by and data-field is the name of the column (other ones has numbers as data-fields by default).